microsetta_private_api/repo/qiita_repo.py

```diff
@@ -1,10 +1,34 @@
+from microsetta_private_api.repo.admin_repo import AdminRepo
 from microsetta_private_api.repo.base_repo import BaseRepo
 from microsetta_private_api.qiita import qclient
 from microsetta_private_api.repo.metadata_repo import retrieve_metadata
 from microsetta_private_api.repo.metadata_repo._constants import MISSING_VALUE
+from microsetta_private_api.repo.survey_answers_repo import SurveyAnswersRepo
 
 
 class QiitaRepo(BaseRepo):
+    def lock_completed_surveys_to_barcodes(self, barcodes):
+        # lock survey-sample association
+        with self._transaction as t:
+            admin_repo = AdminRepo(t)
+            sar_repo = SurveyAnswersRepo(t)
```

Comment:
```suggestion
        admin_repo = AdminRepo(self._transaction)
        sar_repo = SurveyAnswersRepo(self._transaction)
```

Fixed the indenting on the rest of the function accordingly and it should work.

Reply:
That did it, thank you